.github/workflows/R-CMD-check.yaml

```diff
@@ -0,0 +1,90 @@
+# Workflow derived from https://github.com/r-lib/actions/tree/v2/examples
+# Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
+on:
+  push:
+    branches: [main, master]
+  pull_request:
+    branches: [main, master]
+  workflow_dispatch:
+
+name: R-CMD-check.yaml
+
+permissions: read-all
+
+jobs:
+  R-CMD-check:
+    runs-on: ${{ matrix.config.os }}
+
+    name: ${{ matrix.config.os }} (${{ matrix.config.r }})
+
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - {os: macos-latest,   r: 'release'}
+          - {os: windows-latest, r: 'release'}
+          - {os: ubuntu-latest,   r: 'devel', http-user-agent: 'release'}
+          - {os: ubuntu-latest,   r: 'release'}
+          - {os: ubuntu-latest,   r: 'oldrel-1'}
+
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+      R_KEEP_PKG_SOURCE: yes
+
+    steps:
+
+      - uses: actions/checkout@v4
+
+      - uses: r-lib/actions/setup-pandoc@v2
+
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          r-version: ${{ matrix.config.r }}
+          http-user-agent: ${{ matrix.config.http-user-agent }}
+          use-public-rspm: true
+
+      - uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          extra-packages: any::rcmdcheck reticulate
+          needs: check
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
+
+      # - name: setup r-birdnet venv
+      #   shell: Rscript {0}
+      #   run: |
+
+      #     path_to_python <- reticulate::virtualenv_create(
+      #       envname = "r-birdnet",
+      #       python = Sys.which("python"), # placed on PATH by the setup-python action
+      #       packages = c(
+      #         "numpy", "pandas"
+      #       )
+      #     )
+
+      #     writeLines(sprintf("RETICULATE_PYTHON=%s", path_to_python),
+      #                Sys.getenv("GITHUB_ENV"))
+
+
+      - name: setup r-birdnet venv and install birdnet
+        shell: Rscript {0}
+        run: |
+
+          reticulate::install_python("3.11")
```

Comment:
wasn't python 3.11 already setup in line 53?

Reply:
Agreed! Line 75 was removed.